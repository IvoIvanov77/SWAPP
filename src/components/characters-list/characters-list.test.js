// import React from 'react';
// import { shallow } from 'enzyme';
// import ChatactersList from './characters-list.component';

it('should render ChatactersList component', () => {
  //   expect(shallow(<ChatactersList />)).toMatchSnapshot();
});
