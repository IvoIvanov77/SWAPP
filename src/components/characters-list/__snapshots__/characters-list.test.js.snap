// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ChatactersList component with hasNextPage = false should render ChatactersList component 1`] = `ShallowWrapper {}`;

exports[`ChatactersList component with hasNextPage = true should render ChatactersList component 1`] = `ShallowWrapper {}`;
